### PR TITLE
DSC19-177 add gherkin tests

### PR DESCRIPTION
**Pull request by MattyJ007**

### CODE REVIEW

File: features/resources/lab-result-organization2.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "resourceType": "Organization",
+  "identifier": {
+    "value": "12345678"
+  }
+}
```
Comment (bradsawadye) [inline features/resources/lab-result-organization2.json]:
```suggestion
}

```

File: features/resources/lab-result-organization1.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "resourceType": "Organization",
+  "name": "KEMRI Clinic123"
```
Comment (MattyJ007) [inline features/resources/lab-result-organization1.json]: The name here doesn't match the name listed in the questionnaireResponse. How is the search working :|

File: features/case-report.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Given('that the practitioner sending the reports exists', ensurePractitionerExists)
+
+When('a case report is sent through', sendCovid19CaseReport)
+
+Then('the case report should be stored in the fhir server', verifyCovid19CaseReportInFhir)
+
+Then('it should be also be available in dhis', verifyCovid19CaseReportInDhis)
+
+Then('it should then be deleted in both fhir and dhis', cleanupCovid19CaseReport)
```
Comment (MattyJ007) [inline features/case-report.js:23]: There is a Location resource that exists after the tests complete. Can this not be deleted?

File: features/case-report.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  verifyCovid19CaseReportInFhir,
+  cleanupCovid19CaseReport,
+  ensurePractitionerExists
+} = require('./utils')
+
+setDefaultTimeout(30000)
+
+Given('that the practitioner sending the reports exists', ensurePractitionerExists)
+
+When('a case report is sent through', sendCovid19CaseReport)
```
Comment (MattyJ007) [inline features/case-report.js:17]: The Questionnaires are not cleaned up post test - if this is simple could you please remove them

**Review by MattyJ007 — COMMENTED** [review]
Good work - everything is running on mine 👍🏼 Just a few comments

File: features/case-report.js (hunk 4 above)
Comment (bradsawadye) [reply features/case-report.js:17]: This because we do the IG importation

File: features/case-report.js (hunk 3 above)
Comment (bradsawadye) [reply features/case-report.js:23]: sorted

File: features/resources/lab-result-organization1.json (hunk 2 above)
Comment (bradsawadye) [reply features/resources/lab-result-organization1.json]: Sorted

**Review by MattyJ007 — COMMENTED** [review]
LGTM

**Pull request merged**